blog/2021-04-30-descript-case-study.md

```diff
@@ -0,0 +1,86 @@
+
+Even though the migration was prototyped with a self hosted cluster of Temporal (using our Kubernetes helm chart), they decided to sign up for Temporal Cloud as an early design partner. As a small startup, they didn't want to manage Temporal themselves.
+"We migrated progressively using feature flags, and verified performance as we went, but we had confidence in the Temporal team so we didn't stress about load testing."
+
+## Looking Back
+
+Temporal has helped Descript solve fundamental reliability issues with its core transcription loop. The frequency of production incidents has declined from once-a-week to virtually zero. Nicolas attributes this success to the ability to test everything with unit tests as well as normal end-to-end testing on a staging environment.
+
+Testing is important, but Temporal opened up more opportunities since migration:
+
+- Orchestration of multiple Workflows: you can flexibly reuse a workflow once it is written, allowing the transcription workflow to be easily composed with other APIs. This helped Descript develop a major transcription correction feature much more quickly than with the previous architecture.
+- Specialized Compute for Machine Learning: Some parts of the transcription process use Python components that generate CUDA instructions - these must be run on workers with GPUs, and then reliably merged back into a general data processing workflow. Temporal's ability to route activities to multiple Task Queues made this straightforward.
+- Session Routing for Stateful Execution: You can finetune the execution of sequential activities on the same worker, using Go SDK Sessions, which Descript will need (but doesn't yet use).
+
+An unexpected advantage that Descript is enjoying is better observability of workflows, which helps improve customer service even when Descript is not at fault.
+
+> "When a customer reports an issue, it's very easy for us to just put the Workflow ID into the Temporal Web UI to see what is going on... I can see a summary of workflow status and access the full event history for audit needs. "
+
+
+## What's Next?
+
+Now that Descript has had production experience running with Temporal Cloud, more opportunities are presenting themselves in a diverse array of use cases, including:
+
+- billing and subscription management
+- other CPU intensive backend workloads
+- the (fan favorite) Identify Speaker feature
+
+Other teams at Descript are waiting for our NodeJS client to explore migrating other forms of workloads.
+
+> "Everyone in the company can see more and more use cases where Temporal would be useful."
+
+
+## Working with Temporal
+
+
+Descript has found Temporal’s Slack very helpful. It can be difficult to adopt a core infrastructure technology like Temporal so early without help from the community and Temporal's core developers. It has also helped make the case internally to see other well regarded engineering teams ask similar questions and adopt alongside Descript.
```

Comment:
The "It can be difficult to adopt a core infrastructure technology like Temporal so early without help from the community and Temporal's core developers" sentence is a little awkward, maybe just drop "so early"?

Reply:
```suggestion
Descript has found Temporal’s Slack very helpful. It can be difficult to adopt a core infrastructure technology like Temporal without help from the community and Temporal's core developers. It has also helped make the case internally to see other well regarded engineering teams ask similar questions and adopt alongside Descript.
```